src/View/Product/show.html.twig

```diff
@@ -1,10 +1,23 @@
 {% extends 'layout.html.twig' %}
+{% from "_macros/ratingStars.html.twig" import ratingStars %}
 
-{% block stylesheet %}<link rel="stylesheet" href="../assets/css/show.css">{% endblock %}
+{% block stylesheet %}
+    <link rel="stylesheet" href="../assets/css/style.css">
```

Comment:
is style the global css file ? if it is please remove it from this block cause it is already injected inside the layout